Makefile

```diff
@@ -24,20 +24,43 @@ build: ## 🔨 Build the Application
 	./scripts/set_python_env.sh
 	npm run build
 
+setup: ## Setup proposals and generate an initial key
+	@echo -e "\e[34m$@\e[0m" || true
+	CCF_PLATFORM=${CCF_PLATFORM} ./scripts/kms_setup.sh --network-url "${KMS_URL}"  --certificate_dir "${KEYS_DIR}"
+
+stop-host:  ## 🏃 Stop the host
+	@echo -e "\e[34m$@\e[0m" || true
+	sudo lsof -t -i :8000 | xargs -r sudo kill -9
+
+stop-idp:  ## 🏃 Stop the idp
+	@echo -e "\e[34m$@\e[0m" || true
+	sudo lsof -t -i :3000 | xargs -r sudo kill -9
+	
+stop-all: stop-host stop-idp # Stop all services
+	@echo -e "\e[34m$@\e[0m" || true
+	
+# idp commands to issue JWT
+start-idp:  ## 🏃 Start the idp for testing jwt
+	@echo -e "\e[34m$@\e[0m" || true
+	cd test/utils/jwt && nohup npm run start  &
```

Comment:
[nit] I think it would be more convenient if you can output stdout and stderror to a file when something is going wrong

Reply:
Maybe something like:

```
IDP_LOG_DIR_PATH ?= ${WORKSPACE}/idp_log.txt
cd test/utils/jwt && nohup npm run start > ${IDP_LOG_DIR_PATH} 2>&1 &
```

Reply:
BTW WORKSPACE needs to be exported for this command as well like you do for other make file commands

Reply:
Ah, I didn't know that thanks.
Then can you just export WORKSPACE env var please?

Without setting it manually, currently nohup.out says:

```
An error occurred: Error: ENOENT: no such file or directory, open 'undefined/cert.pem' 
```

Reply:
[311cbc5](https://github.com/microsoft/azure-privacy-sandbox-kms/pull/14/commits/311cbc5cf693fbf0d9d07ee2e38442edfaafd1c1)